### PR TITLE
Add support for direct_import parameter to skip objects reloading

### PR DESCRIPTION
**Pull request by dalthon**
Just a rebase of https://github.com/toptal/chewy/pull/637 with added spec and a small fix regarding id types

### CODE REVIEW

File: lib/chewy/type/adapter/orm.rb [hunk 1]
```diff
@@ -45,6 +45,7 @@ def identify(collection)
         # Import options:
         #
         #   <tt>:batch_size</tt> - import batch size, 1000 objects by default
+        #   <tt>:direct_import</tt> - use the collectiov values as-is, instead of reloading
```
Comment (rabotyaga) [inline lib/chewy/type/adapter/orm.rb:48]: The wording in https://github.com/sideqik/chewy/commit/21df05b77eb8ec72cf09811ea4dcd8919080c4f6 seems more clear to me

File: lib/chewy/type/import.rb [hunk 2]
```diff
@@ -67,6 +67,7 @@ module ClassMethods
         # @option options [String] suffix an index name suffix, used for zero-downtime reset mostly, no suffix by default
         # @option options [Integer] bulk_size bulk API chunk size in bytes; if passed, the request is performed several times for each chunk, empty by default
         # @option options [Integer] batch_size passed to the adapter import method, used to split imported objects in chunks, 1000 by default
+        # @option options [Boolean] direct_import passed to the adapter import method, used skip the collection reload
```
Comment (rabotyaga) [inline lib/chewy/type/import.rb:70]: The wording in https://github.com/sideqik/chewy/commit/21df05b77eb8ec72cf09811ea4dcd8919080c4f6 seems more clear to me

File: spec/chewy/type/adapter/active_record_spec.rb [hunk 3]
```diff
@@ -86,7 +86,19 @@ def import(*args)
...
-      specify { expect(import(cities)).to eq([{index: cities}]) }
+      specify do
+        cities
+        expects_db_queries do
+          expect(import(cities, direct_import: false)).to eq([{index: cities}])
+        end
+      end
+      specify do
+        cities
+        expects_no_query do
```
Comment (rabotyaga) [inline spec/chewy/type/adapter/active_record_spec.rb:97]: Does it really fail if some db queries are made? Doesn't seem so.

**Review by rabotyaga — CHANGES_REQUESTED** [review]

File: spec/chewy/type/adapter/active_record_spec.rb (hunk 3 above)
Comment (dalthon) [reply spec/chewy/type/adapter/active_record_spec.rb:97]: Changed! Now it does! While linting I removed the raise keyword by mistake

File: lib/chewy/type/import.rb [hunk 4]
```diff
@@ -67,6 +67,7 @@ module ClassMethods
         # @option options [String] suffix an index name suffix, used for zero-downtime reset mostly, no suffix by default
         # @option options [Integer] bulk_size bulk API chunk size in bytes; if passed, the request is performed several times for each chunk, empty by default
         # @option options [Integer] batch_size passed to the adapter import method, used to split imported objects in chunks, 1000 by default
+        # @option options [Boolean] import objects without reloading
```
Comment (rabotyaga) [inline lib/chewy/type/import.rb:70]:
```suggestion
        # @option options [Boolean] direct_import skips object reloading in ORM adapter
```

Comment (rabotyaga) [reply lib/chewy/type/import.rb:70]: or
```suggestion
        # @option options [Boolean] direct_import import objects without reloading
```

**Review by rabotyaga — COMMENTED** [review]
Should we add the option to the `Chewy::Type::IMPORT_OPTIONS_KEYS` ? I think otherwise it would be impossible to use it in `default_import_options`.

**Comment by dalthon** [comment]
@rabotyaga I was checking values at `Chewy::Type::IMPORT_OPTIONS_KEYS` then I realized that those are not matching with method `import` options:

```diff
< consistency
> direct_import
< pipeline
< raw_import
< refresh
< replication
> parallel
> suffix
> update_failover
> update_fields
```

Do you think that the same applies for `parallel`, `suffix`, `update_failover` and `update_fields`?

**Comment by rabotyaga** [comment]
@dalthon `refresh`, `consistency`, `replication` & `pipeline` belong to `Routine`: https://github.com/toptal/chewy/blob/7efd2e34b2a01cc2b4f72f9bb0a076825787cc36/lib/chewy/type/import/routine.rb#L25-L30
`raw_import` is passed to form adapter.
Let's leave missing `import` keys like `update_failover` & `update_fields` since nobody complains.

And let's add CHANGELOG entry.

**Comment by dalthon** [comment]
@rabotyaga changelog added

**Pull request merged**